packages/patternfly-4/react-core/src/components/Accordion/AccordionToggle.js

```diff
@@ -3,35 +3,31 @@ import PropTypes from 'prop-types';
 import { css } from '@patternfly/react-styles';
 import { AngleRightIcon } from '@patternfly/react-icons';
 import styles from '@patternfly/patternfly/components/Accordion/accordion.css';
-import { Button } from '@patternfly/react-core';
```

Comment:
Why did we switch this to not use a PF4 button?

Reply:
Okay saw @mcoker comment for this.  Can we delete the commented out Button for pf4?

Reply:
Absolutely. :)